neurons/_miner/miner_session.py

```diff
@@ -207,21 +211,26 @@ def queryZkProof(self, synapse: protocol.QueryZkProof) -> protocol.QueryZkProof:
             bt.logging.debug("Model session created successfully")
             synapse.query_output, proof_time = model_session.gen_proof()
             model_session.end()
-            try:
-                bt.logging.info("✅ Proof completed \n")
-            except UnicodeEncodeError:
-                bt.logging.info("Proof completed \n")
+            bt.logging.info("Proof completed \n")
```

Comment:
May be more ideal to keep the check up here due to the try catch - total response time logs are displayed regardless of whether the proof process succeeded, so the check mark may provide an impression of success even in cases where the process failed